src/f77_interface.F

```diff
@@ -26,6 +26,8 @@ MODULE f77_interface
    USE cell_methods,                    ONLY: init_cell
    USE cell_types,                      ONLY: cell_type
    USE cp2k_info,                       ONLY: get_runtime_info
+   USE cp_dlaf_utils,                   ONLY: cp_dlaf_finalize,&
```

Comment:
all of this should de decorated with #ifdef statements....

Reply:
Same question as above: I followed what it is done with ELPA, where the functions still exist and can be called, but then result in `CPABORT`. Do you want to go to a different route, where there is no DLA-Future related code in CP2K if `-DCP2K_USE_DLAF=off`?